src/aws/lambda/utils.ts

```diff
@@ -0,0 +1,55 @@
+import * as aws from '@cdktf/provider-aws';
+import { Construct } from "constructs";
+import { IEnvironmentVars } from './interface'
+
+// import { resolve } from 'path';
```

Comment:
Is sonar running on this repo? Something like this should be picked up there or in linting rules during the pre commit.

Reply:
Hi @jamescrowley321 
I dont think sonar is there in this repo although i have remove all the commented code, any thing else needs to be change in above?

Reply:
Let's try it out.

Reply:
Update the README and add a few examples we can use to test and then we will merge.